.gitignore

```diff
@@ -64,8 +64,11 @@ target/
 #Ipython Notebook
 .ipynb_checkpoints
 
-# vscode config
+# IDEs config (except templates)
 .vscode/*
+!.vscode/*.template.json
+.idea/
```

Comment:
what's `idea`?

Reply:
you have "no idea" what idea is?  :-D   It is the config folder for jetbrains tools . I have "no idea" why i added it :-)